src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCount.java

```diff
@@ -53,12 +72,17 @@ public double toAltAlleleFraction() {
     }
 
     /**
-     * Returns the minor-allele fraction.
-     * @return      alternate-allele fraction
+     * Returns the maximum likelihood estimaate of the minor-allele fraction.
+     *
+     * The likelihood of a alt reads and r ref reads is proportional to f^a(1-f)^r + f^r(1-f)^a, where f in [0,1/2] is
```

Comment:
sounds good!  i guess that the outlier probability doesn't affect the per-het max likelihood MAF anyway.